lib/plugins/rotate_asg_instances.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
Do we need this?

Reply:
This is community-accepted code style.  I think we should leave it.